lib/connections/structs.go

```diff
@@ -93,7 +93,15 @@ func (c internalConn) Type() string {
 }
 
 func (c internalConn) Transport() string {
-	return c.connType.Transport()
+	transport := c.connType.Transport()
+	ip := net.ParseIP(c.LocalAddr().String())
```

Comment:
I don't think this works, as `LocalAddr().String()` should include a port number. (If you think it does, a test to prove it would be sweet.)

Reply:
As usual, this is Well Tested(TM)

Reply:
Apart from, like, not working, it looks good to me.